doc/source/_includes/rllib_we_are_hiring.rst

```diff
@@ -0,0 +1,5 @@
+.. admonition:: We're hiring!
```

Comment:
i think there's another hiring post (not just rllib) somewhere?

Reply:
yes, in the "cluster" part.

Reply:
to be clear, I grouped all messaging into one folder now so that we can consolidate it in follow-up PRs. the "rllib" hiring thingy is specific to rllib right now, but should it be? maybe. we could establish some best practices here. Also, Asawari and co. might find this setup easier to announce Ray summits etc. more easily.